src/tasks/keycloak/keycloak.ts

```diff
@@ -47,17 +47,37 @@ const errors: string[] = []
 const keyCloakRealm = 'otomi'
 
 async function main(): Promise<void> {
-  if (!env.isDev) await waitTillAvailable(env.KEYCLOAK_ADDRESS)
-
+  await waitTillAvailable(env.KEYCLOAK_ADDRESS)
   const keycloakAddress = env.KEYCLOAK_ADDRESS
   const basePath = `${keycloakAddress}/admin/realms`
   let token: TokenSet
+  const isHttps = env.KEYCLOAK_ADDRESS.startsWith('https://')
+  if (isHttps && process.env.NODE_EXTRA_CA_CERTS) {
+    // unfortunately openid client does not seem to honor NODE_EXTRA_CA_CERTS
```

Comment:
I am not sure, that would be also the case for harbor and gitea but we did not observe that, did we?
Would you like to reproduce it together?

Reply:
removed as it was unneeded